Sources/Mantis/CropView/CropView.swift

```diff
@@ -37,6 +37,9 @@ let hotAreaUnit: CGFloat = 32
 let cropViewPadding:CGFloat = 14.0
 
 class CropView: UIView {
+
+    public var config = Mantis.Config()
```

Comment:
CropView is a internal class so I think property should not be public. Also I suggest passing a DialConfig to CropView.init, so we may not need this property here.

Reply:
true